src/ontology/mondo-edit.obo

```diff
@@ -334863,15 +334863,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 
 [Term]
 id: MONDO:0014598
-name: developmental and epileptic encephalopathy, 31
-def: "Any early infantile epileptic encephalopathy in which the cause of the disease is a mutation in the DNM1 gene." [MONDO:patterns/disease_series_by_gene]
+name: developmental and epileptic encephalopathy, 31A
+def: "Any developmental and epileptic encephalopathy in which the cause of the disease is a heterozygous mutation in the DNM1 gene." [https://orcid.org/0000-0002-0587-4693, MONDO:patterns/disease_series_by_gene, OMIM:616346]
 subset: gard_rare {source="GARD:16094", source="MONDO:GARD"}
 subset: nord_rare {source="MONDO:NORD"}
 subset: rare
 synonym: "DEE31" BROAD ABBREVIATION [OMIM:616346]
+synonym: "DEE31A" EXACT [OMIM:616346]
 synonym: "developmental and epileptic encephalopathy 31" BROAD [OMIM:616346, OMIM:genemap2]
+synonym: "Developmental and epileptic encephalopathy 31A, autosomal dominant" EXACT [https://orcid.org/0000-0001-9310-0163]
 synonym: "DNM1 early infantile epileptic encephalopathy" EXACT [MONDO:design_pattern, MONDO:patterns/disease_series_by_gene]
-synonym: "DNM1-Encephalopathy and Neurodevelopmental Disorder" BROAD []
+synonym: "DNM1-Encephalopathy and Neurodevelopmental Disorder" BROAD [https://orcid.org/0000-0001-9310-0163]
```

Comment:
```suggestion
synonym: "DNM1-encephalopathy and neurodevelopmental disorder" BROAD [https://orcid.org/0000-0001-9310-0163]
```

Reply:
@yshwetar did you see the ICBO tutorial yesterday where I demonstrated this? I _think_ you should be able to click 'commit suggestion' above. But it has not worked for me in the past to do these inline commits on large ontology files, so it may not work. 

But the synonym above should be lowercase, except for the gene name

Reply:
I may have missed bits as I had to step away - but I think I did it correctly!

Reply:
there is a video on slide 104 about inline commits: https://docs.google.com/presentation/d/17VRGBPloup9XhtujMYc47wqqdHxjJa5rV0EMtGK8vJo/edit#slide=id.g2e9753b3d69_1_182

Reply:
your revisions are correct, thanks!